package.json

```diff
@@ -41,6 +41,6 @@
     "cover": "cover run jasmine-node spec && cover report html && opener cover_html/index.html"
   },
   "engines": {
-    "node": ">=0.6.0"
+    "node": ">=4"
```

Comment:
I wouldn't consider this a breaking change because `node >=4` was already required by `mime` dependency:

```
% docker run node:0.8 npm install q-io
npm WARN engine mime@1.6.0: wanted: {"node":">=4"} (current: {"node":"0.8.28","npm":"2.13.3"})
q-io@1.13.6 node_modules/q-io
```